tests/c/idempotent_outline.c

```diff
@@ -24,12 +24,12 @@
 //     --- End aot ---
 //     --- Begin jit-pre-opt ---
 //     ...
-//     %{{_}}: i{{size}} = call @idem1(%{{_}}, %{{_}}) <idem_const 10i{{size}}>
+//     %{{_}}: i{{size}} = call @idem1(%{{_}}, %{{_}}) <idem_const 28i{{size}}>
 //     ...
 //     --- End jit-pre-opt ---
 //     --- Begin jit-post-opt ---
 //     ...
-//     %31: i32 = call @fprintf(%{{_}}, %{{_}}, 4i{{size}}, 10i{{size}})
+//     %31: i32 = call @fprintf(%{{_}}, %{{_}}, 4i{{size}}, 28i{{size}})
```

Comment:
By the way, the expected output in this test looked wrong to me. I'll investigate this after, as there might be a (separate) bug in our idempotent promotes.

Reply:
I am not sure I understand how this PR can change the idem_const. That feels like this PR (or the ykllvm PR) *might* itself introduce a new bug?

Reply:
I actually think the opposite. In "untickles" an existing bug.

Manual inspection of the test, I think `28` is the correct value to promote. Do you agree?